pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "sml-catalogue"
-version = "0.2.1-rc.1"
+version = "0.3.0-rc.1"
```

Comment:
just to note, amending this from a preprod rollback test (this is where I noticed it was using the old build files and creating new releases instead of just deploying)